base/exports.jl

```diff
@@ -1210,8 +1210,11 @@ export
 
 # multiprocessing
     addprocs,
+    amap,
+    amap!,
     ClusterManager,
     fetch,
+    imap,
```

Comment:
`amap(f, c...; kv...) = collect(imap(f, c...; kv...))`, and `amap` is subjected to the same tests as `map`, ...